tests/src/main/java/io/openliberty/tools/eclipse/test/it/LibertyPluginSWTBotMavenTest.java

```diff
@@ -964,6 +966,45 @@ public void testLaunchConfigurationMatching() {
         LibertyPluginTestUtils.validateLibertyServerStopped(projectPath.toAbsolutePath().toString() + "/target/liberty");
 
     }
+    
+    /**
+     * Tests the Clean project option provided under liberty run configuration option. Test will check 
+     * if the application has started and also will check for the presence of project clean and dev mode commands 
+     * from the console tab 
+     */
+    
+    @Test
+    public void testRunCleanProjectCheckbox() {
+
+        // Delete any previously created configs.
+        deleteLibertyToolsRunConfigEntriesFromAppRunAs(MVN_APP_NAME);
+
+        // Launch "Run Configurations" window and check "Clean project"
+        launchDashboardAction(MVN_APP_NAME, DashboardView.APP_MENU_ACTION_START_CONFIG);
+        Shell shell = getRunConfigurationsShell();
+        checkRunCleanProjectCheckBox(shell, MVN_APP_NAME);
+
+        // No need to run here. Just Apply and Close.
+        go("Apply", shell);
+        go("Close", shell);
+
+        // Start dev mode. This should start locally.
+        launchDashboardAction(MVN_APP_NAME, DashboardView.APP_MENU_ACTION_START);
+
+        LibertyPluginTestUtils.validateApplicationOutcome(MVN_APP_NAME, true, projectPath.toAbsolutePath().toString() + "/target/liberty");
+        //Reads the text from the console output tab
+        String consoleText =LibertyPluginTestUtils.getConsoleOutput();
+        Assert.isTrue(consoleText.contains("clean io.openliberty.tools:liberty-maven-plugin:dev"));//checks if the consoleText contains the maven clean command 
```

Comment:
We should use org.junit.jupiter.api.Assertions instead of the "Assert" Eclipse API